src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -2048,6 +2048,9 @@
         - name: --undrainable-node-behavior
           type: string
           short-summary: Define the behavior for undrainable nodes during upgrade. The value should be "Cordon" or "Schedule". The default value is "Schedule".
+        - name: --localdns-config
+          type: string
+          short-summary: Set the localDNS Profile for a nodepool with a JSON config file.
```

Comment:
May I ask if `localdns` is a proper noun? If not, it is recommended to separate them
```suggestion
        - name: --local-dns-config
          type: string
          short-summary: Set the local DNS Profile for a nodepool with a JSON config file.
```

Reply:
Yes! localDNS is a proper noun and should be joined this way :D